dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -641,6 +643,7 @@ def test_frame_series_arithmetic_methods():
             assert_eq(l.sub(r, axis=axis), el.sub(er, axis=axis))
             assert_eq(l.mul(r, axis=axis), el.mul(er, axis=axis))
             assert_eq(l.div(r, axis=axis), el.div(er, axis=axis))
+            assert_eq(l.divide(r, axis=0), el.divide(er, axis=0))
```

Comment:
This part of the test is checking that multiple values for `axis=` are properly supported. Updating this line to `axis=axis` should ensure that things `divide` is working as expected

```suggestion
            assert_eq(l.divide(r, axis=axis), el.divide(er, axis=axis))
```